code/modules/mob/living/simple_animal/friendly/drone/drone_tools.dm

```diff
@@ -0,0 +1,84 @@
+
+/obj/item/storage/drone_tools
+	name = "built-in tools"
+	desc = "Access your built in tools"
+	icon = 'icons/hud/screen_drone.dmi'
+	icon_state = "tool_storage"
+	item_flags = ABSTRACT
+	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | UNACIDABLE | ACID_PROOF
+
+/obj/item/storage/drone_tools/Initialize()
+	. = ..()
+	ADD_TRAIT(src, TRAIT_NODROP, ABSTRACT_ITEM_TRAIT)
+
+/obj/item/storage/drone_tools/ComponentInitialize()
+	. = ..()
+	var/static/list/drone_builtins = list(	/obj/item/crowbar/drone, /obj/item/screwdriver/drone, /obj/item/wrench/drone, \
+											/obj/item/weldingtool/drone, /obj/item/wirecutters/drone)
+	var/datum/component/storage/STR = GetComponent(/datum/component/storage)
```

Comment:
https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-descriptive-and-obvious-names